modules/extensions-spring-support/src/test/java/org/mule/module/extension/internal/capability/xml/SchemaGeneratorTestCase.java

```diff
@@ -51,6 +52,10 @@
 public class SchemaGeneratorTestCase extends AbstractMuleTestCase
 {
 
+    @Parameterized.Parameter(0)
+    public Class<?> extensionUnderTest;
+    @Parameterized.Parameter(1)
```

Comment:
add separation line when fields are annotated. That helps readability quite a bit